packages/migration/run-migrations.sh

```diff
@@ -8,6 +8,8 @@
 #
 # Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
 
+set -e # fail if any of the commands fails
```

Comment:
^, was there a reason to not fail the whole script if a single migration fails?

Reply:
@naftis not sure but E2E test environment deploy fails if you add this. 

```
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | $ /app/node_modules/.bin/migrate-mongo up --file ./build/dist/src/migrate-mongo-config-openhim.js
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | $ /app/node_modules/.bin/migrate-mongo status --file ./build/dist/src/migrate-mongo-config-openhim.js
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ┌────────────────────────────────────────────────────────────────┬──────────────────────────┐
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ Filename                                                       │ Applied At               │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20220825071508-channels-v1.0.1.js                              │ 2024-10-01T05:36:00.799Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20220826062206-channels-v1.0.1-to-v1.1.0.js                    │ 2024-10-01T05:36:00.862Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20221011150329-channels-v1.1.0-to-v1.2.0.js                    │ 2024-10-01T05:36:00.882Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20221121124457-remove-old-transactions.js                      │ 2024-10-01T05:36:00.896Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20221124202628-user-audit-channels.js                          │ 2024-10-01T05:36:00.937Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20230131130400-add-certificate-issue-channel.js                │ 2024-10-01T05:36:00.953Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20230210112528-marriage-channels.js                            │ 2024-10-01T05:36:01.027Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20230328101901-potential-duplicate.js                          │ 2024-10-01T05:36:01.042Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20230822142113-rename-request-correction-to-make-correction.js │ 2024-10-01T05:36:01.059Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20231115134531-remove-validation-and-update-channels.js        │ 2024-10-01T05:36:01.132Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20231117171009-remove-register-channels.js                     │ 2024-10-01T05:36:01.152Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ├────────────────────────────────────────────────────────────────┼──────────────────────────┤
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | │ 20240305193551-remove-openhim-collections.js                   │ 2024-10-01T05:36:01.171Z │
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | └────────────────────────────────────────────────────────────────┴──────────────────────────┘
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | $ /app/node_modules/.bin/migrate-mongo up --file ./build/dist/src/migrate-mongo-config-application-config.js
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | Migration - Remove config and informantsmsnotifications successful
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | ERROR: Could not migrate up 20240708170106-remove-config-collections.js: not authorized on ocrvs-5932-migration__application-config to execute command { dropDatabase: 1, lsid: { id: UUID("90d81825-f997-413b-93b7-4857a92fccc0") }, $clusterTime: { clusterTime: Timestamp(1727762034, 1), signature: { hash: BinData(0, 1B3D4C6D97FE350243664E687C99C280133BCE3B), keyId: 7416731534561902597 } }, $db: "ocrvs-5932-migration__application-config" } MongoServerError: not authorized on ocrvs-5932-migration__application-config to execute command { dropDatabase: 1, lsid: { id: UUID("90d81825-f997-413b-93b7-4857a92fccc0") }, $clusterTime: { clusterTime: Timestamp(1727762034, 1), signature: { hash: BinData(0, 1B3D4C6D97FE350243664E687C99C280133BCE3B), keyId: 7416731534561902597 } }, $db: "ocrvs-5932-migration__application-config" }
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at Connection.onMessage (/app/node_modules/mongodb/lib/cmap/connection.js:231:30)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at MessageStream.<anonymous> (/app/node_modules/mongodb/lib/cmap/connection.js:61:60)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at MessageStream.emit (node:events:517:28)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at processIncomingData (/app/node_modules/mongodb/lib/cmap/message_stream.js:125:16)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at MessageStream._write (/app/node_modules/mongodb/lib/cmap/message_stream.js:33:9)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at writeOrBuffer (node:internal/streams/writable:392:12)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at _write (node:internal/streams/writable:333:10)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at Writable.write (node:internal/streams/writable:337:10)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at Socket.ondata (node:internal/streams/readable:809:22)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    |     at Socket.emit (node:events:517:28)
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | error Command failed with exit code 1.
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | error Command failed with exit code 1.
ocrvs-5932-migration_migration.1.0ivd6wxum58u@e2e-dev    | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ocrvs-5932-migration_migration.1.nrtqcsx3lutr@e2e-dev    | error Command failed with exit code 1.
```

Also @Nil20 take a note of this please. I think it's caused by https://github.com/opencrvs/opencrvs-core/blob/develop/packages/migration/src/migrations/application-config/20240708170106-remove-config-collections.ts#L18